acceptance/bundle/python/restricted-execution/mutators.py

```diff
@@ -0,0 +1,16 @@
+from dataclasses import replace
+from databricks.bundles.jobs import Job, Task
+from databricks.bundles.core import job_mutator, Bundle
+import os
+
+
+@job_mutator
+def read_envs(bundle: Bundle, job: Job) -> Job:
+    value = os.getenv("SOME_ENV_VAR", "default")
+    with open("envs.txt", "w") as file:
+        file.write(value)
+
+    tasks = bundle.resolve_variable(job.tasks)
```

Comment:
Why do you modify the job tasks here? Can we just return early keeping the job as is?